example/todo-app/src/main/scala/io/github/shogowada/scalajs/reactjs/example/todoapp/Main.scala

```diff
@@ -26,7 +26,7 @@ class Main {
       override def render() = {
         <.div()(
           <.h3()("TODO"),
-          <.reactElement(new TodoList(), TodoList.Props(items = state.items)),
+          new TodoList()(TodoList.Props(items = state.items)),
```

Comment:
Actually, even if you are using some kinda DI framework, you should be able to do something like below.

```scala
class MyVirtualDOMElements(container: Container) extends VirtualDOMElements {
  def TodoList = container.resolve[TodoList]
}

class TodoApp(<: MyVirtualDOMElements) extends ReactClassSpec {
  def render() = <.div()(
    <.TodoList(TodoList.Props(items = state.items))
  )
}
```

I can write up some tutorial for this approach.